lib/data/fetchFiles.js

```diff
@@ -3,12 +3,15 @@ const fs = require('fs'),
   url = require('url'),
   path = require('path'),
   shell = require('shelljs'),
+  flatten = require('array-flatten'),
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat
?

Reply:
Only from nodejs 11. Do we want to bump required version? @pavelloz

Reply:
nope